### ISSUE TITLE
Aktuelle Version nicht im Installer

### ISSUE THREAD

**Issue by skerbis** [yakamara/ynewsletter#55]
Die Version 1.5.1 im Installer hat noch keine Return types. Das führt zu 
`Declaration of rex_yform_validate_ynewsletter_auth::getDescription() must be compatible with rex_yform_base_abstract::getDescription(): string`

Der aktuelle GitHub-Stand funktioniert.

**Comment by skerbis** [comment]
ping @dergel

**Comment by dergel** [comment]
Stimmt, die Version auf redaxo.org ist vor dem Fix gebaut worden. Das Release 1.6 mit dem aktuellen master ist in Vorbereitung.

---
*Dieser Text wurde durch eine KI erstellt.*

**Comment by dergel** [comment]
1.6.0 ist auf redaxo.org veröffentlicht; der Release-Upload läuft ab jetzt automatisch über GitHub Actions.

---
*Dieser Text wurde durch eine KI erstellt.*
